answers.md

```diff
@@ -1 +1,18 @@
 # Answers Go In This File
+//Shaina 
+//Qustion 1
+1. Git is a tool used by software developers to manage their code. Git keeps track of code by saving existing code. Software developers use it to code with others by branching, merging commits, and adding their code to the main base. Git is a local repository.
+//Question 2
+2. GitHub is like a social media platform, or a version control system, for coders or software developers. It allows software developers to start repositories, fork other repositories, and request pulls. GitHub is a remote repository.
+//Question 3
+3. Commit is a separate branch of a repository. It allows developers to change or add to the main code without changing or affecting the original or main base. Editing code in a commit is different from editing a document or file in a way that if you make a change in a google document or file, it will affect the document or file. Everyone will see the change you made to the original document or file. Commit allows you to make changes you like without affecting the original repository. If you want someone to see your commit and make changes to the original repository, you can request a pull to merge your commit and add it to the original repository. 
+//Question 4
+4. If you want to command git to add a file to a repository, type in git 'git add file.txt' or 'git add file.md'. If you want to add a directory to a repository, type, 'git add directory/'. If you like to add both directory and file, type 'git add .'. The period after 'git add' allows you to add everything to the repository. 
+//Question 5
+5. If you like to upload your local repository from git to GitHub, your remote repository, you’d like to command the computer by typing, “git push”. This allows you to transfer your commit to a remote repository. 
+//Question 6 
+6. If you’d like to get your copy of an existing repository, you’d like to go to GitHub and copy the link. You, then, go to Cloud9 to command the computer to copy the repository by typing “git clone link”. You add the link after “git clone”. This will create a new directory for you to work on separately without making any changes to the main base. 
+//Question 7
+7. To fork, a repository is to make a copy of a repository and make it your own. For example, if you find a code that you like that someone else has and you’d like to do something completely different to it, like work on a project, you’d fork it and make the repository your own without disrupting the original repository. Usually, when you fork a repository, you wouldn’t want to merge it with the original repository but it is possible. 
+//Question 8
+8. To branch, a repository means that you intend to merge your copy with the original repository. While forking, although you can merge the repository with the main base, you rarely intend to do so. It is not necessary and very uncommon.
```

Comment:
I think you're implying this but not saying explicitly that branching makes a copy of the repository. How might you add in that important detail? Also, can you give an example of WHY you might make a branch?

Check out the [collaborative git slides](https://docs.google.com/presentation/d/1h8vaNFbu9oMK0xeV8673UdHiNjouNpusMHbK6Ge_-vw/edit#slide=id.g158fbd3a59f_0_267) for some help.